keyboards/acheron/austin/rules.mk

```diff
@@ -0,0 +1,55 @@
+## chip/board settings
+# the next two should match the directories in
+#  <chibios>/os/hal/ports/$(MCU_FAMILY)/$(MCU_SERIES)
+MCU_FAMILY = STM32
+MCU_SERIES = STM32F0xx
```

Comment:
The majority of the MCU config can now be replaced by a single line: `MCU = STM32F072`. See `quantum/mcu_selection.mk`.

Reply:
Done in #7507

Reply:
Done in https://github.com/qmk/qmk_firmware/pull/7507/commits/49267eb39e62be9ec50210420da0f1028f67ecb5